packages/itwinui-react/src/core/Dialog/DialogMain.test.tsx

```diff
@@ -64,15 +63,14 @@ it('should render with custom style and className', () => {
     </DialogMain>,
   );
 
-  const dialog = container.querySelector(
-    '.iui-dialog.iui-dialog-visible',
-  ) as HTMLElement;
+  const dialog = container.querySelector('.iui-dialog') as HTMLElement;
   expect(dialog).toBeTruthy();
   expect(dialog.classList.contains('test-class')).toBeTruthy();
   expect(dialog).toHaveStyle('color: red');
 });
 
-it('should close on Esc click and move focus back', () => {
+it('should close on Esc click and move focus back', async () => {
+  config.disabled = true;
```

Comment:
that is so weird I could have sworn it was failing without, because after a while I asked Mayank if there's anything else we could do....but I also just tested removing it and now it passed..? So I guess we don't need it lol. But basically it's to disable the react transition time for animation, in this case dialog enter and exit animation time.